corehq/apps/app_manager/suite_xml.py

```diff
@@ -1816,6 +1838,7 @@ def get_target_module(case_type, module_id, with_product_details=False):
                 })
 
         if module.get_app().commtrack_enabled:
+            # TODO add here?
```

Comment:
Don't think we need anything here (similar comment to above about advanced forms)